view/src/components/common/EventParticpantCard/EventParticpantCard.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Box from '@mui/material/Box'
+import { useState, useEffect } from 'react';
+
+interface User {
+  id: number
+  name: string
+  number: number
+  created_at: number
+  updated_at: number
+};
+
+interface Events {
+  id: number
+  name: string
+  description: string
+  max_attendee: number
+  start_at: number
+  end_at: number
+  created_at: number
+  updated_at: number
+  user: User[]
```

Comment:
ここもできればUserの配列なのでusersにしたいですが、バックエンド側の変更が必要なので、自分の作業量と相談して決めてください〜

```suggestion
  users: User[]
```